data/man/sddm.conf.rst.in

```diff
@@ -74,6 +74,13 @@ OPTIONS
 	`/run/netns/mynet`.  Default value is empty.  (The value is ignored if
 	the operating system is not Linux.)
 
+`DefaultSession=`
+	System-wide default session file name. For example, `plasma.desktop`.
+	The file must be already exists in X11 or Wayland `SessionDir`,
+	or it has no effect. Please note that `RememberLastSession` will also
+	set the default session and will override this option.
+	Default value is empty.
+
```

Comment:
If I understand correctly, with `DefaultSession` the user set the default session to be selected the very first time SDDM runs.  This description could use some improvement to be make it clear.

Reply:
Sorry for my poor English. I want to express that if `RememberLastSession=true` and there is already a successful login before, it would take precedence over `DefaultSession`.

Reply:
Here's a suggestion for a description:

```
Defines the session that will get selected by default.
In case `RememberLastSession` is enabled and there is a last session available, this last session will be selected.
```